CHANGES.md

```diff
@@ -6,10 +6,12 @@ Version 0.5.0 is the fifth release.
 
 This release includes some API changes:
 
+`cats.laws.discipline.eq` no longer provides `Eq` instances for `tuple2` and `tuple3`, these instances and together with some other new instances for `tuple`s are now provided by `cats.std.tuple` (through inheriting the instance trait defined in algebra 0.4.2).
```

Comment:
Nitpicky but could you capitalize `Tuple2`, etc.? The formatting and context makes it look like you're referring to real types.